tensorflow/compiler/tf2xla/g3doc/generate_ops_coverage.py

```diff
@@ -0,0 +1,64 @@
+from datetime import date
+from google.protobuf import text_format
+from pytablewriter import MarkdownTableWriter # https://pypi.org/project/pytablewriter/
+import support_pb2 as Support
+
+
+today = date.today()
+today = today.strftime("%Y-%m-%d")
+
+supported_ops = Support.OpSupports()
+
+
+def parse_supported_ops():
+  with open("./support.pbtxt", 'r') as f:
+    return text_format.Parse(f.read(), supported_ops)
+
+
+supported_ops = parse_supported_ops()
+table_name = "TF ops bridges coverage list"
+fieldnames = ["op_name", "old_bridge", "new_bridge"]
+old_bridge = new_bridge = all_bridges = 0
+num_ops = len(supported_ops.support)
+value_matrix = []
+
+for op in supported_ops.support:
+  old_bridge_value = new_bridge_value = ":white_check_mark:"
+  if op.supports_old_bridge:
+    old_bridge += 1
+    old_bridge_value = ":heavy_check_mark:"
+  if op.supports_new_bridge:
+    new_bridge_value = ":heavy_check_mark:"
+    new_bridge += 1
+  if (op.supports_old_bridge and op.supports_new_bridge):
+    all_bridges += 1
+  value_matrix.append([op.graph_op_name, old_bridge_value, new_bridge_value])
+
+writer = MarkdownTableWriter()
+
+writer_ops_stats = MarkdownTableWriter(
```

Comment:
Bringing in a new dependency just to format a table does not quite seem worth it, maybe just do a helper function?

Reply:
Maybe there's something in https://github.com/tensorflow/tensorflow/tree/master/tensorflow/tools/docs

But I think those all generate html tables.

Reply:
The problem now is also where we are going to take the source of truth? As the protos were removed in the meantime.